db/main.py

```diff
@@ -0,0 +1,85 @@
+import base64
+from sqlalchemy import create_engine, Table, MetaData, Column, String
+from sqlalchemy.sql import select
+
+
+class DbRow(object):
+    def __init__(self, url, metadata=None):
+        self.url = url
+        if(metadata is not None):
+            self.rowcount = 1
+            self.metadata = str(metadata._row[0])
+        else:
+            self.rowcount = 0
+            self.metadata = metadata
+
+
+class DbUtils(object):
+
+    # engine - May not need it as a class variable
+    engine = None
+
+    # table - May not need it as a class variable
+    mydata = None
+
+    # connection - May not need it as a class variable
+    connection = None
+
+    def establishDbConnection(self):
```

Comment:
Python convention is underscores to seperate the words on a function name. Rename to `establish_db_connection`